global.json

```diff
@@ -1,5 +1,6 @@
 {
     "sdk": {
-      "version": "5.0.x"
```

Comment:
It actually didn't do what we thought. On my machine it resolved the version `6.0.100-preview.7.21379.14` 🤷

Reply:
argh... surprising